Libraries/LibWeb/HTML/HTMLSelectElement.cpp

```diff
@@ -380,6 +381,13 @@ WebIDL::ExceptionOr<void> HTMLSelectElement::set_value(String const& value)
     return {};
 }
 
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#dom-cva-validity
+GC::Ref<ValidityState const> HTMLSelectElement::validity()
+{
+    auto& realm = this->realm();
+    return realm.create<ValidityState>(realm, dynamic_cast<FormAssociatedElement*>(this));
```

Comment:
I don't believe ValidityState requires a mutable FormAssociatedElement, which would allow all of these `::validity()` helpers to remain const qualified

Reply:
Good point, I forgot to put that back after troubleshooting.
Done.

Reply:
Not a huge deal, but you could also use `as<FormAssociatedElement>(this)` here, which is slightly more compact.